experimental/add2app/SimpleIOSExample/Podfile

```diff
@@ -0,0 +1,26 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+# Added these two lines for Flutter:
+flutter_application_path = '../example_module'
+system("pushd #{flutter_application_path} && flutter create . && popd")
+load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
+
+target 'SimpleIOSExample' do
+  # Comment the next line if you don't want to use dynamic frameworks
```

Comment:
Is this coming from a template?
I feel like we don't need this.

Reply:
Yes, that was auto-generated by `pod init`. I've removed it.

Reply:
Looks like it's still here :)